user-interface/src/case-detail/panels/CaseDetailCourtDocket.scss

```diff
@@ -1,25 +1,33 @@
 @import '../../styles/abstracts/colors';
 
 #case-detail-court-docket-panel {
-  .docket-entry {
-    padding-bottom: 2rem;
+  #searchable-docket {
```

Comment:
Check.  I believe this can be reset since the button is no longer located on this screen and this section no longer scrolls.